cluster/ci/config/prow/config.yaml

```diff
@@ -89,6 +89,7 @@ tide:
     - openshift/origin-web-console
     - openshift/origin-web-console-server
     - openshift/cluster-operator
+    - openshift/os
```

Comment:
We should probably keep private repos in separate queries.

Reply:
We're planning to do a new public repo.  Can an openshift admin execute on making a new `openshift/os-container` git repo?  https://github.com/openshift/os/pull/13#issuecomment-386416253

Reply:
/cc @smarterclayton ^